### PR TITLE
Niels9001/recategorization

### PR DESCRIPTION
**Pull request by niels9001**
Moving a bunch of categories around and:

- Removing `SettingsPageExample.md` and move it to the `SettingsExpander` page as it was only a sample.
- Shadows: overview page, and dedicated CardShadow/DropShadow pages.

This requires: https://github.com/CommunityToolkit/Tooling-Windows-Submodule/pull/133

### CODE REVIEW

File: components/Extensions/samples/TextBoxExtensions.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ The `SurfaceDialOptions` property adds features from the Surface Dial control to
 
 ```xaml
```
Comment (michael-hawker) [inline components/Extensions/samples/TextBoxExtensions.md:126]: We didn't bring Surface Dial options over, did we?

File: components/Extensions/samples/ShadowAnimations.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+category: Animations
+subcategory: Shadows
+discussion-id: 0
+issue-id: 0
+icon: Assets/ShadowAnimation.png
+---
+
+Either type of Attached Shadow can be easily animated using the Toolkit's [`AnimationSet`](../animations/AnimationSet.md) api. These provide an easy XAML based way to animate a wide variety of elements, including a variety of shadow properties. They can also be animated with any other composition animation technique in code-behind as well using either the [`AnimationBuilder`](../animations/AnimationBuilder.md) or built-in composition animations.
+
+> **Platform APIs:** [`BlurRadiusDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.blurradiusdropshadowanimation), [`ColorDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.colordropshadowanimation), [`OffsetDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.offsetdropshadowanimation), [`OpacityDropShadowAnimation`](/dotnet/api/microsoft.toolkit.uwp.ui.animations.opacitydropshadowanimation)
```
Comment (michael-hawker) [inline components/Extensions/samples/ShadowAnimations.md:17]: We should think about these sections as none of these links are going to work for now... 🤔

Unless we replace them with the `CommunityToolkit.WinUI.Animations...` equivalent's but those pages won't be up for a bit still anyway

File: components/Extensions/samples/ListViewExtensions.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ await MyGridView.SmoothScrollIntoViewWithItemAsync(item: object, itemPlacement:
 ### Methods
 
 | Methods | Return Type | Description |
-| -- | -- | -- |
+|---------|-------------|-------------|
```
Comment (michael-hawker) [inline components/Extensions/samples/ListViewExtensions.md:127]: This is an old-style page, we should just re-write.

File: components/Extensions/samples/AttachedShadows.md [hunk 4]
```diff
@@ -1,20 +1,18 @@
...
 issue-id: 0
 icon: Assets/Shadow.png
 ---
 > **Platform APIs:** [`AttachedCardShadow`](/dotnet/api/microsoft.toolkit.uwp.ui.media.attachedcardshadow), [`AttachedDropShadow`](/dotnet/api/microsoft.toolkit.uwp.ui.attacheddropshadow)
 
-## Introduction
-
 There are two types of attached shadows available today, the `AttachedCardShadow` and the `AttachedDropShadow`. It is recommended to use the `AttachedCardShadow` where possible, if you don't mind the dependency on Win2D. The `AttachedCardShadow` provides an easier to use experience that is more performant and easier to apply across an entire set of elements, assuming those elements are rounded-rectangular in shape. The `AttachedDropShadow` provides masking support and can be leveraged in any UWP app without adding an extra dependency.
 
 ### Capability Comparison
```
Comment (michael-hawker) [inline components/Extensions/samples/AttachedShadows.md:18]: These should be ## two-level headers.

Thinking the docs validation is going to yell at us for not having titles at the top, but I can inject the title from the metadata in on the transpose step...

**Review by michael-hawker — COMMENTED** [review]
Quick pass at some of the changes, mostly just taking notes for things left for us to maybe do before, doesn't have to be in this PR.

File: components/Extensions/samples/ShadowAnimations.md (hunk 2 above)
Comment (Arlodotexe) [reply components/Extensions/samples/ShadowAnimations.md:17]: Perhaps we just notify the user that updated docs are coming soon?

**Review by michael-hawker — APPROVED** [review]
Let's go forward!

File: components/Extensions/samples/ShadowAnimations.md (hunk 2 above)
Comment (Arlodotexe) [reply components/Extensions/samples/ShadowAnimations.md:17]: Moved here - https://github.com/CommunityToolkit/Windows/pull/213

File: components/Extensions/samples/TextBoxExtensions.md (hunk 1 above)
Comment (michael-hawker) [reply components/Extensions/samples/TextBoxExtensions.md:126]: Oh, they appear to be here still: https://github.com/CommunityToolkit/Windows/tree/main/components/Extensions/src/Text

Not sure how they work with WASDK... something that probably needs to be tested...

File: components/Extensions/samples/ListViewExtensions.md (hunk 3 above)
Comment (michael-hawker) [reply components/Extensions/samples/ListViewExtensions.md:127]: Have spruced up this page in an upcoming commit to #214. Removed these and added the SmoothScroll as an actual sample.

**Pull request merged**